Source/ASTextNode+Beta.h

```diff
@@ -35,6 +35,12 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (readonly) BOOL usingExperiment;
 
+/**
+ * A Boolean indicating if the text node will expose links within the attributedText
+ * as UIAccessibitlyElements
+ */
+@property (nonatomic) BOOL accessibilityExposeLinksAsAccessibilityElements;
```

Comment:
Is there any case where we'd want to disable this? I'd suggest adding it as an ASExperimental flag, and once tested, we can just globally launch it. It should at least be default-on in the end state, and we could add a property later if anyone needs to disable it.

Reply:
I think we should gracefully start integrating it with this property of `NO` and run an experiment from outside.

Reply:
I think we should just bite the bullet and do the experimental flag – in reality it'll be easier & still safe to just announce that "hey in this release links are accessibility elements and we've already vetted the change!" than it will to say "we changed the default/removed the `accessibilityExposeLinksAsAccessibilityElements` flag so be careful." Let's get to the end state fast.

Reply:
Just pushed a new commit that will introduce the `ASExperimentalExposeTextLinksForA11Y` flag that is by default `NO` and removed the `accessibilityExposeLinksAsAccessibilityElements` flag.